### DISCUSSION TITLE
Disable print statements

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#4842] [Ideas]
Many of our users (Jason, Andarist, ...) have asked for the ability to disable print statements.

They have also asked for the ability to save print statements in one replay and re-apply them in another :)

**Comment by jcmorrow** [comment]
I first read this as "Disable users on my team/in this replay from introducing any print statements", so to clarify for others: this is the scenario where you have a useful print statement, you don't want to lose the expression by deleting it, but you also wish it would stop logging temporarily to make the console easier to deal with.

**Reply by jasonLaster** [reply]
Yep!

**Reply by jasonLaster** [reply]
The design approach i'm thinking about is the 👁️ icon that design tools use which would hide the logs.

**Comment by jonbell-lot23** [comment]
March 15, 2022:

I could also see this leading one day to colour-coding results, adding emojis next to certain ones, etc. Right now we have one approach: show them. But in time we'll need more options like temporarily hiding or increasing the visibility.
